moped-editor/src/components/GridTable/Filters.js

```diff
@@ -123,6 +132,57 @@ const generateEmptyField = (uuid) => {
   return { ...defaultNewFieldState, id: uuid };
 };
 
+/**
+ * Get configuration about the filters using the query parameters present in the URL
+ * @param {Array} filters - filters from the URL {field, operator, value}
+ * @param filtersConfig - The configuration object for the filters
+ * @return {Object}
+ */
+const makeInitialFilterParameters = (filters, filtersConfig) => {
+  if (filters.length === 0) return [];
+
+  return filters.reduce((acc, filter) => {
+    const filterUUID = uuidv4();
```

Comment:
After spending a lot of time in this code, I found that the uuid provides a way to reliably point to and edit an existing filter when there are interactions in the UI. A unique ID is used because the filters are stored in an object, and don't have an index or guaranteed order.

Reply:
+1 - 100%. The refactor that I've been working on for the internal state should help us get away from building that object, remove the need for the uuids, and mirror the shape of the `filters` state. 🤞